compiler/src/depgraph.ts

```diff
@@ -0,0 +1,349 @@
+import { LP, LPNode, LPError, NamedAnd, NulLP } from './lp';
```

Comment:
Since you are breaking this up, do you mind following the pattern `lntoamm` has? Eg create an `ammtoaga` directory, rename `ammtoaga.ts` to `ammtoaga/index.ts` and put this file inside `ammtoaga/` as well?